caravel/assets/javascripts/SqlLab/components/DatabaseSelect.jsx

```diff
@@ -46,6 +43,12 @@ class DatabaseSelect extends React.Component {
 
 DatabaseSelect.propTypes = {
   onChange: React.PropTypes.func,
+  databaseId: React.PropTypes.number,
```

Comment:
I tested this passing databaseId as null, no warnings were shown. According to [https://github.com/facebook/react/issues/2166](url), for proptypes that are not isRequired, the null value means it's intentional, which will be bypassed by React, the undefined value will cause a warning.  For this one I think we don't have to put databaseId as isRequired since it's for passing the value to Select. If null is passed then the Select box is empty, as in default state.